### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by maksimgm**
@bengrunfeld

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+/*html*/
+html { 
+  background: url("http://imagery.playerattack.com/TwitchCon-SanFrancisco.png") no-repeat center center fixed; 
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+/*Header*/
+
```
Comment (bengrunfeld) [inline style.css]: comment has an empty line after it, but the comment before doesn't. Please stay consistent.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/*html*/
```
Comment (bengrunfeld) [inline style.css]: Change to `/* General Page Styles /*`

File: style.css (hunk 1 above)
Comment (bengrunfeld) [inline style.css]: Comment should be `/* Header */` for readability.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+
```
Comment (maksimgm) [inline style.css]: Create a typography section. `h1,h2,h3,h4,h5,h6,p,a { font-family:helvetica; }`

**Comment by maksimgm** [comment]
:+1:

**Pull request merged**